lib/elixir/lib/stream.ex

```diff
@@ -1155,6 +1155,10 @@ defmodule Stream do
   @spec cycle(Enumerable.t) :: Enumerable.t
   def cycle(enumerable)
 
+  def cycle(enumerable) when length(enumerable) == 0 do
+    raise ArgumentError, "Stream cannot cycle over empty enumerable"
```

Comment:
Error messages in Elixir start with lower case. But in this case, it may be correct if you are referencing the `Stream` module, but I would rather say you are referencing the "stream" noun. Long story short: should this message start with lowercase for consistency?